src/deferred.js

```diff
@@ -0,0 +1,13 @@
+
+var oldDeferred = jQuery.Deferred;
+
+jQuery.Deferred = function() {
+	var deferred = oldDeferred.apply( this, arguments );
+
+	deferred.pipe = function() {
```

Comment:
Oh yeah, I totally missed that case! I need to go back and double check some of the edge cases since for example a `Deferred` can have a different object applied as its prototypical `promise` object and none of that is being covered here.